src/components/Categories.js

```diff
@@ -0,0 +1 @@
+// Check Status
```

Comment:
- [ ] Kindly work on the changes requested by the previous code reviewer 🙏

Reply:
Hi! Ashaka, Kindly read the project requirements.

Reply:
> * [ ]  Kindly work on the changes requested by the previous code reviewer 🙏

Hi! Ashaka, Kindly read the project requirements.

Reply:
- [ ] @AlucardSanin I read your comments and checked your code. I believe you should display only a button with text "Check status". Adding a comment doesn't change anything.  Please make this and your project would be ready for approval.

Reply:
Shouldn't you try the app before writing your review? 

![image](https://user-images.githubusercontent.com/99287916/200134573-de471a17-1625-4567-818b-396d1ed22bd1.png)

The button is there, inside the Categories view of the app, I'm saying that the first review was blocking me from approval, and so on until now I wasted my 3 requests for reviews.

Reply:
I created a component and wrote that comment because that's exactly what the first reviewer said: _please create a categories component with the text check status for now._